redisinsight/ui/src/pages/browser/modules/key-details/components/key-details-subheader/KeyDetailsSubheader.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+
+import { EuiButton, EuiCheckbox, EuiFlexGroup, EuiFlexItem } from '@elastic/eui'
+import AutoSizer from 'react-virtualized-auto-sizer'
+
+import VerticalDivider from 'uiSrc/pages/rdi/statistics/components/vertical-divider'
+import { KeyDetailsHeaderFormatter } from '../../../key-details-header/components/key-details-header-formatter'
+import styles from './styles.module.scss'
+
+interface Props {
+  showTtl: boolean
+  onShowTtl: (checked: boolean) => void
+  onAddKey: () => void
+}
+
+export const KeyDetailsSubheader = ({
+  showTtl,
+  onShowTtl,
+  onAddKey
+}: Props) => (
+  <div className={styles.subheaderContainer}>
+    <AutoSizer disableHeight>
+      {({ width = 0 }) => (
+        <div style={{ width }}>
+          <EuiFlexGroup justifyContent="flexEnd">
+            <EuiFlexItem className={styles.keyFormatter__item}>
```

Comment:
no reason use BEM since we use scoped styles,
it is better to name keyFormatterItem